### PR TITLE
modify devops api for tenant

### PR DESCRIPTION
**Pull request by shaowenchen**
**What type of PR is this?**
 /kind feature

**What this PR does / why we need it**:

normalize devops API for tenant 

/cc @wansir

### CODE REVIEW

File: pkg/kapis/devops/v1alpha3/register.go [hunk 1]
```diff
@@ -51,128 +51,120 @@ func AddToContainer(container *restful.Container, devopsClient devopsClient.Inte
 		ws := runtime.NewWebService(GroupVersion)
 		handler := newDevOpsHandler(devopsClient, k8sclient, ksclient, ksInformers, k8sInformers)
 		// credential
-		ws.Route(ws.GET("/workspaces/{workspace}/devopsprojects/{projectName}/credential/").
+		ws.Route(ws.GET("/devops/{devops}/credentials/").
```
Comment (wansir) [inline pkg/kapis/devops/v1alpha3/register.go:54]: suffix `/`  is unnecessary

Comment (shaowenchen) [reply pkg/kapis/devops/v1alpha3/register.go]: fixed

**Pull request merged**